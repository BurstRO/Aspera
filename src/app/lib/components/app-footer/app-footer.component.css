.footer-info {
    padding-left: 0px;
    list-style-type: none;
    white-space: nowrap;
    display: inline;
}

.footer-info > li {
    display: inline;
    color:red;
    font-size: 0.8em;
    margin-left: 0px;
    padding-left: 10px;
}
