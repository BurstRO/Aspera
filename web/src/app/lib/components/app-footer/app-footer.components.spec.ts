import {
    async,
    ComponentFixture,
    TestBed
} from '@angular/core/testing';

import { AppFooterComponent } from './app-footer.component';

describe('AppFooterComponent', () => {
    let component: AppFooterComponent;
    let fixture: ComponentFixture<AppFooterComponent>;
    let dom : any;

    beforeEach(async(() => {
        TestBed.configureTestingModule({
            declarations: [ AppFooterComponent ]
        });
        fixture = TestBed.createComponent(AppFooterComponent);
        component = fixture.componentInstance;
    }));

    test('should exist', () => {
        expect(component).toBeDefined();
    });

    test('should be rendered correctly', () => {

        expect(fixture).toMatchSnapshot();
    });
});
