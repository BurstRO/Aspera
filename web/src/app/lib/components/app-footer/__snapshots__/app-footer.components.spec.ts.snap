// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooterComponent should be rendered correctly 1`] = `
<app-footer>
  <footer
    class="main-footer"
  >
    <div>
      <ul
        class="footer-info"
      >
        <li>
          <a
            href="https://burst-coin.org"
            target="_blank"
          >
            <i
              aria-hidden="true"
              class="fa fa-globe"
            />
             Website
          </a>
        </li>
        <li>
          <a
            href="https://explore.burst.cryptoguru.org/"
            target="_blank"
          >
            <i
              aria-hidden="true"
              class="fa fa-database"
            />
             Block Explorer
          </a>
        </li>
        <li>
          <a
            href="https://twitter.com/burstcoin_dev"
            target="_blank"
          >
            <i
              aria-hidden="true"
              class="fa fa-twitter"
            />
             Twitter
          </a>
        </li>
        <li>
          <a
            href="https://github.com/PoC-Consortium"
            target="_blank"
          >
            <i
              aria-hidden="true"
              class="fa fa-comment"
            />
             Discord
          </a>
        </li>
        <li>
          <a
            href="https://github.com/PoC-Consortium"
            target="_blank"
          >
            <i
              aria-hidden="true"
              class="fa fa-github"
            />
             Github
          </a>
        </li>
      </ul>
    </div>
  </footer>
</app-footer>
`;
